src/main/java/io/cdap/plugin/gcp/dataplex/common/util/DataplexUtil.java

```diff
@@ -392,4 +392,21 @@ public static MetadataServiceClient getMetadataServiceClient(GoogleCredentials c
     return metadataServiceClient;
   }
 
+  /**
+   * Get fully-qualified name (FQN) for a Dataplex Entity
+   * (FQN format: dataplex:{projectId}.{location}.{lakeName}.{zoneName}).
+   *
+   * @param datasetProject Name of the dataplex project
+   * @param location       location Name
+   * @param lake           Name of the lake associated with the asset
+   * @param zone           Name of the zone associated with the asset
+   * @param tableName      Name of the table associated with the asset
+   * @return String fqn
+   */
+  public static String getFQN(String datasetProject, String location, String lake, String zone, String tableName) {
```

Comment:
Not sure to keep/remove it. We may need it in future, but we don't need it right now.